website/common/locales/en/achievements.json

```diff
@@ -147,5 +147,8 @@
   "achievementPolarProModalText": "You collected all the Polar Pets!",
   "achievementPlantParent": "Plant Parent",
   "achievementPlantParentText": "Has hatched all standard colors of Plant pets: Cactus and Treeling!",
-  "achievementPlantParentModalText": "You collected all the Plant Pets!"
+  "achievementPlantParentModalText": "You collected all the Plant Pets!",
+  "achievementDinosaurDynasty": "Dinosaur Dynasty",
+  "achievementDinosaurDynastyText": "Has hatched all standard colors of bird and dinosaur pets: Falcon, Owl, Parrot, Peacock, Penguin, Rooster, Pterodactyl, T-Rex, Triceratops, and Velociraptor!",
+  "achievementDinosaurDynastyModalText": "You colleted all the bird and dinosar pets!"
```

Comment:
Typo "dinosar"

Reply:
oh, also "colleted" XD

Reply:
Fixed. You know, English didn't used to have standard spelling. I'm just saying. XD